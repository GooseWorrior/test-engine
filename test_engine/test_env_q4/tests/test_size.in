newDefault 1
at 1 0
[] 1 0
at 1 0
at 1 1 
[]= 1 0 1
at 1 0
[]= 1 1 2
at 1 0
at 1 1
at 1 -1
at 1 2
erase 1 0
at 1 0
at 1 1 
at 1 2
erase 1 1
at 1 0
at 1 1
[]= 1 5 5
[]= 1 3 3
[]= 1 4 4
[]= 1 2 2
at 1 2
at 1 3
at 1 4
at 1 5
newInit 2
at 2 1
at 2 2 
at 2 7

