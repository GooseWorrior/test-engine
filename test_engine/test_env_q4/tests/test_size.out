1:
Exception
1:
0
1: (0,0)
0
1: (0,0)
Exception
1: (0,0)
1: (0,1)
1
1: (0,1)
1: (0,1) (1,2)
1
1: (0,1) (1,2)
2
1: (0,1) (1,2)
Exception
1: (0,1) (1,2)
Exception
1: (0,1) (1,2)
1: (1,2)
Exception
1: (1,2)
2
1: (1,2)
Exception
1: (1,2)
1:
Exception
1:
Exception
1:
1: (5,5)
1: (3,3) (5,5)
1: (3,3) (4,4) (5,5)
1: (2,2) (3,3) (4,4) (5,5)
2
1: (2,2) (3,3) (4,4) (5,5)
3
1: (2,2) (3,3) (4,4) (5,5)
4
1: (2,2) (3,3) (4,4) (5,5)
5
1: (2,2) (3,3) (4,4) (5,5)
1: (2,2) (3,3) (4,4) (5,5)
2: (1,2) (3,4) (5,6) (7,8)
2
1: (2,2) (3,3) (4,4) (5,5)
2: (1,2) (3,4) (5,6) (7,8)
Exception
1: (2,2) (3,3) (4,4) (5,5)
2: (1,2) (3,4) (5,6) (7,8)
8
1: (2,2) (3,3) (4,4) (5,5)
2: (1,2) (3,4) (5,6) (7,8)
